web/controllers/user_controller.ex

```diff
@@ -11,6 +11,22 @@ defmodule CoursePlanner.UserController do
     render(conn, "index.html", users: Users.all())
   end
 
+  def show(%{assigns: %{current_user: %{role: role}}} = conn, %{"id" => id}) do
+    if role === "Coordinator" do
```

Comment:
If you need help with `permissions.ex`, drop me a word, it's pattern matching galore so it might confusing at first sight.